python/lsst/ap/association/association.py

```diff
@@ -47,6 +48,18 @@ class AssociationConfig(pexConfig.Config):
         default=1.0,
     )
 
+    trailedSourceFilter = pexConfig.ConfigurableField(
+        target=TrailedSourceFilterTask,
+        doc="Subtask to filter artifact candidates based on morphological "
+            "criteria, i.g. those that appear to be streaks.",
```

Comment:
I don't think this is "artifact candidates"; the only thing that task does is remove long trails per DMTN-199.

```suggestion
        doc="Subtask to remove long trailed sources based on catalog source morphological measurements.",
```

Reply:
Changed doc string to suggested